book/section3/didcomm.md

```diff
@@ -2,11 +2,92 @@
 
 ## Overview
 
-**DIDComm v2** (*Decentralized Identifier Communication version 2*) is a set of communication protocols designed to enable secure, private, and interoperable messaging between DIDs. These protocols have been adopted throughout Identus to standardize many important interactions, like sending and receiving encrypted messages between a mediator its peers.
+**DIDComm** (*Decentralized Identifier Communication*) represents a crucial advancement in the Self-Sovereign Identity ecosystem, providing the secure messaging layer that enables DIDs to interact with one another. The current specification, **DIDComm v2**, evolved from earlier iterations to address the growing needs for privacy, security, and interoperability in digital identity communications.
```

Comment:
dunno if we should call this an `advancement`, maybe it's a `component` or just `DIDComm (Decentralized Identifier Communication) provides a secure messaging layer that enables....`